src/durationinterface.cpp

```diff
@@ -80,6 +80,8 @@ Fraction DurationInterface::GetInterfaceAlignmentDuration(int num, int numBase)
     if (this->HasNum()) num *= this->GetNum();
     if (this->HasNumbase()) numBase *= this->GetNumbase();
 
+    // Fraction duration(noteDur);
+    // duration = duration * numBase / num;
     Fraction duration = DUR_MAX / pow(2.0, (double)(noteDur - 2.0)) * numBase / num;
```

Comment:
I am wondering how these calculations behave with the `Fraction` class.  Is `DUR_MAX` of type `Fraction`?    
 I am thinking it is a sort of enumeration behaving link `int`.  Otherwise the calculations will be using `double`.   Preferably the `int` version of `pow(2, x)` would be used instead of the double version `pow(2.0, x)`, but in any case the `pow()` function will be outputting -0.5 for `pow(2, -1)` in any case.  I am wondering what `- 2.0` is doing.  Converting to whole-note units?

Reply:
Not sure but I think you were looking at an outdated version. This has been remove and uses now the `Fraction(data_DURATION)` constructor.